src/beamlime/handlers/detector_data_handler.py

```diff
@@ -126,3 +129,68 @@ def get(self) -> sc.DataArray:
 
     def clear(self) -> None:
         self._det.clear_counts()
+
+
+# Note: Currently no need for a geometry file for NMX since the view is purely logical.
+# DetectorHandlerFactory will fall back to use the detector_number configured in the
+# detector view config.
+_registry = {
+    'geometry-dream-2025-01-01.nxs': 'md5:91aceb884943c76c0c21400ee74ad9b6',
+    'geometry-loki-2025-01-01.nxs': 'md5:e2e48c30ad02dcfa940b7c26885216f2',
+}
```

Comment:
Why do we have the date in the file name instead of using the `version` ...?

Reply:
Good question! My though was that we actually need multiple at the same time, depending on what date we use for runs. Will not happen in production but in dev/testing. In other words: The old files never become invalid. Instead they will always remain valid for a certain date range. Thus the pooch version mechanism is not what we want here.